llvm/cmake/modules/TableGen.cmake

```diff
@@ -191,9 +191,13 @@ macro(add_tablegen target project)
 
   # FIXME: Quick fix to reflect LLVM_TABLEGEN to llvm-min-tblgen
   if("${target}" STREQUAL "llvm-min-tblgen"
-      AND NOT "${LLVM_TABLEGEN}" STREQUAL ""
-      AND NOT "${LLVM_TABLEGEN}" STREQUAL "llvm-tblgen")
-    set(${project}_TABLEGEN_DEFAULT "${LLVM_TABLEGEN}")
```

Comment:
I suppose `${LLVM_TABLEGEN}` is `llvm-tblgen` by default.
You can modify it if you want to use out-of-tree `llvm-tblgen`.

Reply:
We don't use an out-of-tree `llvm-tblgen`.  For me, on input to this block, `${LLVM_TABLEGEN}` is of the form:
`C:\path\to\dir-within-build-dir\bin\llvm-tblgen.exe`

Reply:
Again, `${LLVM_TABLEGEN}` is `llvm-tblgen` by default.